tests/tools/plugins/ssl_hook_test.cc

```diff
@@ -332,7 +332,7 @@ setup_callbacks(TSHttpTxn txn, int preaccept_count, int client_hello_count, int
   }
   for (i = 0; i < sni_count; i++) {
     cb = TSContCreate(&CB_SNI, nullptr);
-    TSContDataSet(cb, (void *)(intptr_t)i);
+    TSContDataSet(cb, (void *)static_cast<intptr_t>(i));
```

Comment:
I'm surprised this doesn't apply a C++ cast for the (void *) c-style cast. Do you know why it doesn't?

Reply:
Don't know